File: src/main/java/thkoeln/dungeon/restadapter/GameDto.java
package thkoeln.dungeon.restadapter;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.*;
import thkoeln.dungeon.game.domain.GameStatus;

import java.util.UUID;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
@JsonIgnoreProperties(ignoreUnknown = true)
public class GameDto {
    private UUID gameId;
    private GameStatus gameStatus;
    private Integer currentRoundCount;
}

File: src/main/java/thkoeln/dungeon/restadapter/GameDto.kt
package thkoeln.dungeon.restadapter

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import lombok.*
import thkoeln.dungeon.game.domain.GameStatus
import java.util.*

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
@JsonIgnoreProperties(ignoreUnknown = true)
data class GameDto (
    var gameId: UUID? = null,
    var gameStatus: GameStatus? = null,
    var currentRoundCount: Int? = null,
)